### PR TITLE
Chore: added subcategory events under resource

### PR DESCRIPTION
**Pull request by ayushh1804**
## Fixes Issue
Fixed issue #636 Under the Resources section, add another row Hackathons and include relevant information

## Screenshots
image

### CODE REVIEW

**Review by rupali-codes — CHANGES_REQUESTED** [review]
@ayushh1804 it doesn't match with the category, please put this under _other_ category

**Review by rupali-codes — COMMENTED** [review]
Also, seems like your base branch is not updated, please update it with `linkshub:main`

File: database/resources/events.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  {
+    "name": "Hackathon",
+    "description": "This guide provides information regarding what is a Hackathon and how does it works.",
+    "url": "https://tips.hackathon.com/article/what-is-a-hackathon",
+    "category": "resources",
+    "subcategory": "events"
+  },
+  {
+    "name": "Your First Hackathon: A Survival Guide",
+    "description": "Getting started for a hackathon? Here is an article for your reference.This guide provides the best practices for your first Hackathon.",
```
Comment (CBID2) [inline database/resources/events.json:11]: [Proofreading] 
```suggestion
    "description": "Are you attending your first hackathon? Check out this article! It provides tips to help you get started.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @ayushh1804! I just commented on a description that needs improvement. 😀

**Comment by ayushh1804** [comment]
> Hi @ayushh1804! I just commented on a description that needs improvement. 😀

done...thanks for the suggestion😊

**Comment by ayushh1804** [comment]
> @ayushh1804 it doesn't match with the category, please put this under _other_ category

done

**Review by CBID2 — APPROVED** [review]
Lgtm

**Review by rupali-codes — APPROVED** [review]
thanks, looks good to me

**Review by CBID2 — APPROVED** [review]
LGTM!

**Pull request merged**